stream_alert/alert_processor/outputs.py

```diff
@@ -392,7 +390,7 @@ def dispatch(self, **kwargs):
         # Extracting context data to assign the incident
         rule_context = kwargs['alert'].get('context', {})
         if rule_context:
-            rule_context = rule_context[self.__service__]
+            rule_context = rule_context.get(self.__service__, {})
```

Comment:
Could we break out the logic below into a function that returns a tuple of the `user_to_assign` and the `assigned_value`? just trying to understand this flow better